src/util/adb.js

```diff
@@ -418,3 +418,16 @@ export default class ADBUtils {
     });
   }
 }
+
+export async function listADBDevices(adbBin?: string): Promise<Array<string>> {
+  const adbClient = new ADBUtils({adbBin});
+  return await adbClient.discoverDevices();
```

Comment:
Nit, no need to `await` before returning, the promise returned by a call to this async function would still only be resolved when `adbClient.discoverDevices()` is resolved.